pages/en/lb4/Testing-your-application.md

```diff
@@ -42,19 +43,17 @@ integrates these packages and makes them easy to use together with LoopBack.
 
 ### Set up testing infrastructure with LoopBack CLI
 
-{% include note.html content="The LoopBack CLI does not yet support LoopBack 4,
-so using the CLI is not an option with the beta release.
-" %}
-
-<!--
-If you are just getting started with LoopBack, use the LoopBack command-line tool (CLI)  `loopback-cli`. It's ready to use and installs simply by `npm install -g loopback-cli`. You don't need to do any extra steps for setup, and can head straight to the [next section](Testing-Your-Application#acceptance-testing.html).
--->
+LoopBack applications that have been generated by `lb4 app` or `lb4`
```

Comment:
LoopBack application that have been generated using the `lb4 app` command from `@loopback/cli` come with `@loopback/testlab` and `mocha` as a default, so no other testing infrastructure setup is needed.

^ I tried to make it shorter and more to the point. Don't think we need to list both cli commands, just the one we promote as a default for creating a new project in examples. And don't think we need to mention it's a prompt. They should know that because they would've already created the app by now.